tests/UnitBLAS/Level3.hs

```diff
@@ -151,6 +204,22 @@ unitTestLevel3BLAS = testGroup "BLAS Level 3 tests " [
     ,testCase "dgemm on 2x9^T and 2x9 all 1s (column oriented)" $ matmatTest8DGEMM Matrix.SColumn
     ,testCase "cgemm on 2x2 all 1s" matmatTest1CGEMM
     ,testCase "zgemm on 2x2 all 1s" matmatTest1ZGEMM
+
```

Comment:
Why do some of the descriptions below say "gemm" instead of "symm"? it seems like they're all testing the *symm code?